src/Sylius/Behat/Page/Shop/Checkout/AddressPage.php

```diff
@@ -164,7 +164,7 @@ public function getItemSubtotal(string $itemName): string
 
         $subtotalTable = $this->getElement('checkout_subtotal');
 
-        return $subtotalTable->find('css', sprintf('#sylius-item-%s-subtotal', $itemSlug))->getText();
+        return $subtotalTable->find('css', sprintf('[data-test-sylius-item-%s-subtotal]', $itemSlug))->getText();
```

Comment:
I'd go here with `data-test-item-subtotal="item/slug"` attribute:
 - Removed `sylius`, not really needed
 - Slug as a attribute value, because HTML attributes names can only contain a restricted set of characters